local-proxy/package.json

```diff
@@ -7,9 +7,9 @@
   "description": "",
   "main": "dist/index.js",
   "scripts": {
-    "test": "echo \"No test specified\"",
+    "build": "rm -rf dist/ && tsc",
     "lint": "tslint -c ../common/tslint.yml -p .",
-    "build": "rm -rf dist/ && tsc"
+    "test": "ava -v dist/test/test_*.js"
```

Comment:
Nit: Can we just `ava dist/test/*.js`?  snake_caseIsForPython, I thought.

Reply:
Normalized as previously discussed in office, tests live in `src/test` and their format must be `/.*\.(test|spec)\.ts$/`.